server/routes/locations.test.js

```diff
@@ -0,0 +1,38 @@
+const request = require('supertest')
+const { appWithAllRoutes } = require('./testutils/appSetup')
+
+const locationService = {
+  getActiveAgenciesByType: jest.fn(),
+}
+
+let app
+
+beforeEach(() => {
+  app = appWithAllRoutes({ locationService })
+  locationService.getActiveAgenciesByType.mockResolvedValue([])
+})
+
+afterEach(() => {
+  jest.resetAllMocks()
+})
+
+describe('GET /change-prison', () => {
+  test('should display page title', () => {
+    return request(app)
+      .get(`/report/-19/change-prison`)
+      .expect('Content-Type', /html/)
+      .expect(res => {
+        expect(res.text).toContain('What prison did the use of force take place in?')
+      })
+  })
+
+  test('should contain the Continue and Cancel buttons', () => {
+    return request(app)
+      .get(`/report/-19/change-prison`)
+      .expect('Content-Type', /html/)
+      .expect(res => {
+        expect(res.text).toContain('Continue')
```

Comment:
not sure this explicitly requires a separate test? You would want a test for submitting - to ensure the passed agency id is saved in the service

Reply:
I wanted to check that all the components that should be on a page were actually on there. 
Test removed. 
Will do cypress integration test for submission.